src/citrine/citrine.py

```diff
@@ -38,3 +40,11 @@ def projects(self) -> ProjectCollection:
     def users(self) -> UserCollection:
         """Return the collection of all users."""
         return UserCollection(self.session)
+
+    @property
+    def teams(self) -> TeamCollection:
+        """Returns a resource representing all visible teams."""
+        if self.session._accounts_service_v3:
+            return TeamCollection(self.session)
+        else:
+            raise AccountsV3Exception("Teams are not available, please continue using projects")
```

Comment:
Just curious about how this flag will work in practice -- so when a Python SDK user secretly knows about and wants to use these v3 features, would they manually initialize the session with `_account_service_v3 = True`?

Reply:
This flag gets updated every time the user's token is refreshed by a call to the product api's config endpoint. A particularly motivated user can definitely update that private property, but it wouldn't do them any good since the backend services wouldn't have been updated to understand v3 calls.

Reply:
Aaah okay, that makes more sense. Thank you!